packages/components/molecules/f-skeleton-loader/src/components/skeletons/RestaurantCarousel.vue

```diff
@@ -0,0 +1,73 @@
+<template>
+    <div :class="$style['c-restaurantCarouselWrapper']">
+        <heading :class="$style['c-restaurantCarousel-heading']" />
+        <text-block :class="$style['c-restaurantCarousel-desc']" />
+        <div :class="$style['c-restaurantCarousel']">
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
+        </div>
+    </div>
```

Comment:
Delayed comment as I'm just going back through additions.

I'd amend the classname structure here a little bit, so it better matches our conventions:

```html
    <div :class="$style['c-restaurantCarousel']">
        <heading :class="$style['c-restaurantCarousel-heading']" />
        <text-block :class="$style['c-restaurantCarousel-desc']" />
        <div :class="$style['c-restaurantCarousel-list']">
            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
            <restaurant-carousel-card :class="$style['c-restaurantCarousel-card']" />
        </div>
    </div>
```

Reply:
Updated in 0.5.0 PR